### DISCUSSION TITLE
Daily Time Lapse Problem - No recording on 2nd day and no upload

### DISCUSSION THREAD

**Discussion by PixelMafiaProductions** [gopro/labs#359] [General]
Hey guys! 

So i use this template on one of our GoPRo 10 Black:
https://gopro.github.io/labs/control/dailytl/

GoPro is located at a customer who wants to get a documentation of his construction site. 

The cam is installed on a roof and i need to climb up every time if i want to check the status via Quik App as connection mode via BLE sets to off while using QR Code.

Is it anyhow possible to activate connection mode and the daily timelapse video mode alltogether?

Thanks in advance :)

**Comment by dnewman-gpsw** [comment]
Not really. If you use the upload via WiFi, you can monitor the progress via the GoPro cloud.  However, scripting the connection of the app is not possible.

**Reply by PixelMafiaProductions** [reply]
Thanks for that quick response :) 

Autoupload is enabled so i will see this evening first results. Thanks buddy

**Reply by PixelMafiaProductions** [reply]
So while the checking the cam today, i have the problem that the upload didnt work (which cause can be bad wifi of course) and which is more terrific in my opinion, it didnt start recording today properly :( 

I did a reset and fresh qr code and now gonna check tomorrow for the logs and will upload them.
